scripts/pre_push_hook.py

```diff
@@ -328,6 +329,14 @@ def start_linter(files):
     return task.returncode
 
 
+def start_mypy_checks():
+    """Starts the mypy type checks."""
+    task = subprocess.Popen(
+        [PYTHON_CMD, '-m', MYPY_TYPE_CHECK_MODULE, '--skip-install'])
```

Comment:
@DubeySandeep Is it possible to do the same for linters too - i.e. include a skip install flag in the commands list itself, and do the installing third party libs part separately in main (or wherever correct). I thought this would be helpful as we wouldn't have to depend on linter run to ensure all third party libs are installed for mypy before it can run.
@hardikkat24 Did you explore this?

Reply:
@sajalasati In pre-push hook before mypy checks, we check for backend python lib inconsistencies. See [here](https://github.com/oppia/oppia/blob/4172bcc457b5fff70bb8ee70e596b0c7940b2fdb/scripts/pre_push_hook.py#L499)
So do you think that is enough?
And after that if there are files to lint, then linter is run with installs all backend python libs.
Thanks!

Reply:
> @sajalasati In pre-push hook before mypy checks, we check for backend python lib inconsistencies. See [here](https://github.com/oppia/oppia/blob/4172bcc457b5fff70bb8ee70e596b0c7940b2fdb/scripts/pre_push_hook.py#L499)
> So do you think that is enough?
> And after that if there are files to lint, then linter is run with installs all backend python libs.
> Thanks!

@hardikkat24 As per the function documentation, I don't think it's installing any dependencies, it's just checking for inconsistencies.

Reply:
@sajalasati I think that if the backend libraries are not installed fully, the script will stop executing due to inconsistencies. Am I missing something here?